.github/workflows/publish-draft-releases.yml

```diff
@@ -340,11 +340,405 @@ jobs:
       #   run: |
       #     jq --slurp '. | add' ${{ github.workspace }}/extrinsic-benchmark-pallet-manta-pay-*.json > ${{ github.workspace }}/extrinsic-benchmark-pallet-manta-pay.json
 
+  build-node-old:
+    needs:
+      - start-node-builder-old
+      - check-for-runtime-upgrade
+    runs-on: ${{ needs.start-node-builder-old.outputs.runner-label }}
+    env:
+      CARGO_TERM_COLOR: always
+    steps:
+      -
+        if: ${{ needs.check-for-runtime-upgrade.outputs.runtime-version != needs.check-for-runtime-upgrade.outputs.runtime-version-old }}
```

Comment:
These conditionals need to be on each step, unfortunately.
There's no other way to exit early with success.
We need jobs to rely on the runtime upgrade stuff only if the runtime upgrade actually happens.